lib/stream.js

```diff
@@ -44,39 +44,39 @@ Stream.Stream = Stream;
 // Internal utilities
 try {
   const types = require('util').types;
-  if (types && typeof types.isUint8Array === 'function') {
-    Stream._isUint8Array = types.isUint8Array;
+  if (types && typeof types.isArrayBufferView === 'function') {
+    Stream._isArrayBufferView = types.isArrayBufferView;
   } else {
     // This throws for Node < 4.2.0 because there's no util binding and
     // returns undefined for Node < 7.4.0.
-    Stream._isUint8Array = process.binding('util').isUint8Array;
+    Stream._isArrayBufferView = process.binding('util').isArrayBufferView;
   }
 } catch (e) {
 }
 
-if (!Stream._isUint8Array) {
-  Stream._isUint8Array = function _isUint8Array(obj) {
-    return Object.prototype.toString.call(obj) === '[object Uint8Array]';
-  };
+if (!Stream._isArrayBufferView) {
+  // Cached to make sure no userland code can tamper with it.
+  const isArrayBufferView = ArrayBuffer.isView;
+  Stream._isArrayBufferView = isArrayBufferView;
```

Comment:
Oh, in this case we don't actually need to do any caching. I think we can just do `Stream._isArrayBufferView = isArrayBufferView = ArrayBuffer.isView;`

Reply:
done.